documentation/Who-Is-MrChecker/Test-Framework-Modules/DevOPS-Test-Module-Continuous-Delivery-CD.asciidoc

```diff
@@ -12,14 +12,14 @@ CD from Jenkins point of view does not change a lot from Continuous Integration
 
 For Jenkins CD setup please use the same Jenkins settings as for CI https://github.com/devonfw/devonfw-testing/wiki/continuous-integration#jenkins-configuration[link]. The only difference is:
 
-* What type of test you we will execute. Before we have been picking test case(s), however now we will choose test suite(s)
-* Who will will trigger given Smoke/Integration/Performance job
-* What is the name of official branch. This branch ought to use be used always in every CD execution. It will be either *master* or *develop*.
+* What type of test we will execute. Before we have been picking test case(s), however, now we will choose test suite(s)
+* Who will trigger given Smoke/Integration/Performance job
+* What is the name of official branch. This branch ought to use be used in every CD execution. It will be either *master* or *develop*.
 
 === Jenkins for Smoke Tests
 
-In point where we input test name - $TESTNAME ( https://github.com/devonfw/devonfw-testing/wiki/continuous-integration#initial-configuration[link] ), please input test suite which merge by tags -( https://github.com/devonfw/devonfw-testing/wiki/tags-and-test-suites[link] ) how to all test cases need to run only smoke tests.
```

Comment:
how to all test cases need to run only smoke tests. = what is this supposed to mean?

Reply:
I think it means: In variable $TESTNAME, we can set testname or testsuite. So if we want to run only smoke tests,  we can define smoke tags in tests, define testsuite with filtering for this smoke tag and this testsuite put in this $TESTNAME variable. In such way we will run all smoke tests.